cherryontech/unmatched
# #UM-13 tag filter

## Description


This pull request adds a color change and check mark functionality to the tag button.
A user can only select one tag at a time
## Related Tickets & Documents


## Acceptance Criteria
- [x] Filter options are displayed below the search bar.
- [x] Filter tags color change when clicked
- [ ] When I select one or more filters, the displayed profiles are narrowed down to match the selected criteria.
- [ ] Clearing the filters or selecting a new filter updates the search results accordingly.
- [ ] The number of results displayed adjusts based on the applied filters.




## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|   x  | :sparkles: New feature     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings



## What I learned


## Testing steps
Switch to this branch with the following command: git switch feature/UM-13-tag-filter
Install dependencies with the following command: npm i
Run the following scripts: npm run dev to start Vite's dev build & npm run tailwind to start the Tailwind CSS build
View the app in browser (the npm run dev script will return the url you can see the app at)
Navigate to the the Resources page and click on each tag to see added feature


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
